go/ct/rlz/conditions.go

```diff
@@ -669,7 +669,7 @@ func (c *isAddressWarm) Check(s *st.State) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	return s.Accounts.IsWarm(NewAddress(key)), nil
+	return s.Accounts.IsWarm(key.Bytes20be()), nil
```

Comment:
This is weakening readability. I would instead update the `NewAddress` implementation to use `Bytes20be()`. This may also be beneficial for other places where `NewAddress` is used.

Also, the effect of this change seems to be noise ... at least on my system there is no significant difference:
```
cpu: Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz
                                   │   old.log   │              new.log               │
                                   │   sec/op    │   sec/op     vs base               │
Condition_IsAddressWarmCheckWarm-4   41.70n ± 8%   42.02n ± 4%       ~ (p=0.684 n=10)
Condition_IsAddressWarmCheckCold-4   16.64n ± 6%   16.57n ± 2%       ~ (p=0.739 n=10)
geomean                              26.34n        26.39n       +0.20%

```